homeassistant/components/youtube/strings.json

```diff
@@ -47,8 +47,14 @@
           "published_at": { "name": "Published at" }
         }
       },
-      "subscribers": { "name": "Subscribers" },
-      "views": { "name": "Views" }
+      "subscribers": {
+        "name": "Subscribers",
+        "native_unit_of_measurement": "subscribers"
+      },
+      "views": {
+        "name": "Views",
+        "native_unit_of_measurement": "views"
+      }
```

Comment:
Does it not also make sense to drop the `native_unit_of_measurement` in the entity description for these two sensors?